web/themes/joinup/prototype/sass/_drupal-hacks.sass

```diff
@@ -175,3 +175,28 @@ textarea
     &:before
       +icon-size(15px)
       content: '\e617'
+
+// Additional styling for dropbutton.
+.dropbutton-action
```

Comment:
a11y attributes are still missing.
The button is inaccessible with tabbing:
![jul 28 2017 1-51 pm](https://user-images.githubusercontent.com/1923476/28716188-09d01f72-739c-11e7-921e-b93c7c83193b.gif)